tests/test_hermes.rs

```diff
@@ -19,6 +19,12 @@ fn test_react_native_hermes() {
         ("input.js", 2, 0, None)
     );
     assert_eq!(sm.get_original_function_name(11857), Some("<global>"));
+
+    assert_eq!(
+        sm.lookup_token(0, 11947).unwrap().to_tuple(),
+        ("module.js", 1, 4, None)
```

Comment:
This basically points to the `throw` token here (which is at line 2:5) https://github.com/getsentry/rust-sourcemap/blob/2fb5b56c812e7de0981e96c0dbd556c80ee44d64/tests/fixtures/react-native-hermes/module.js#L2